taxi/views.py

```diff
@@ -55,16 +58,22 @@ class ManufacturerDeleteView(LoginRequiredMixin, generic.DeleteView):
 class CarListView(LoginRequiredMixin, generic.ListView):
     model = Car
     paginate_by = 5
-    queryset = Car.objects.all().select_related("manufacturer")
+    queryset = Car.objects.all().select_related(
```

Comment:
No need for all() here. Also why do you prefetch drivers? I don't see you using drivers on car list template